localstack/services/infra.py

```diff
@@ -338,7 +338,7 @@ def terminate_all_processes_in_docker():
     print("INFO: Received command to restart all processes ...")
     cmd = (
         'ps aux | grep -v supervisor | grep -v docker-entrypoint.sh | grep -v "bin/localstack" | '
-        "grep -v localstack_infra.log | awk '{print $1}' | grep -v PID"
+        "grep -v localstack_infra.log | awk '{print $2}' | grep -v PID"
```

Comment:
Fix awk command to get pids correctly from `ps aux` result.

Reply:
great catch